### PR TITLE
Add page titles

### PR DESCRIPTION
**Pull request by luontola**
#1211 & #1212

### CODE REVIEW

File: src/clj/rems/home.clj [hunk 1]
```diff
@@ -30,7 +28,7 @@
 
 (defn- markdown-page [filename]
```
Comment (opqdonut) [inline src/clj/rems/home.clj]: this will conflict with #1246

File: resources/translations/en.edn [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {:t
  {:about {:text [[:div.document
-                  [:h2 "REMS-Demo"]
```
Comment (opqdonut) [inline resources/translations/en.edn]: the translation changes will conflict with #1246

File: src/cljs/rems/atoms.cljs [hunk 3]
```diff
@@ -78,6 +79,17 @@
 (defn enrich-user [user]
   (assoc user :display (str (:name user) " (" (:email user) ")")))
 
+(defn document-title [title]
```
Comment (opqdonut) [inline src/cljs/rems/atoms.cljs]: should this also produce the h1 tag?

File: test/clj/rems/test_browser.clj [hunk 4]
```diff
@@ -89,13 +89,13 @@
...
   (click-navigation-menu "Catalogue")
-  (wait-visible *driver* {:tag :h2, :fn/text "Catalogue"})
+  (wait-visible *driver* {:tag :h1, :fn/text "Catalogue"})
   (wait-page-loaded)
   (screenshot *driver* (io/file reporting-dir "catalogue-page.png")))
 
 (defn go-to-applications []
   (click-navigation-menu "Applications")
-  (wait-visible *driver* {:tag :h2, :fn/text "My Applications"})
+  (wait-visible *driver* {:tag :h1, :fn/text "My Applications"})
```
Comment (opqdonut) [inline test/clj/rems/test_browser.clj]: should you assert the title somewhere here?

**Comment by Macroz** [comment]
I happened to do a small part of this already in #1244 so small changes are required.

**Pull request merged**